app.js

```diff
@@ -1,90 +1,156 @@
 let fs = require('fs'),
```

Comment:
I know you already have a test for the following:
`node app.js --overwrite --files test.csv`
But I think you should add a test for the following:
`node app.js --files test.csv --overwrite`
Because this was a bug I brought up before and you have fixed it but without any coverage